validator/xsd/schedule.xml.xsd

```diff
@@ -242,4 +242,17 @@
     </xs:simpleContent>
   </xs:complexType>
 
+  <xs:simpleType name="optoutEnum">
```

Comment:
My understanding of recording and streaming is that both are independent from each other. As a user I would expect to see these flags in the schedule being presented as separate icon or similar. Is that possible with the enum or should it be split up?

Reply:
For the XML schedule variant: I think we should try to combine both...

Reply:
What is the reasoning for combining both?